earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -302,38 +302,44 @@ ()
 	for (AssetStorage::VersionList::const_iterator v
                 = asset->versions.begin();
              v != asset->versions.end(); ++v) {
-            ${name}AssetVersionD version(*v);
-            // Load an asset version without caching since we may not need it
-            // (offline, obsolete and so on).
-            version.LoadAsTemporary();
-            if ((version->state != AssetDefs::Offline) &&
-                (version->inputs == boundInputs) &&
-                ::IsUpToDate(config_, version->config)) {
+            try {
+              ${name}AssetVersionD version(*v);
+              // Load an asset version without caching since we may not need it
+              // (offline, obsolete and so on).
+              version.LoadAsTemporary();
+              if ((version->state != AssetDefs::Offline) &&
+                  (version->inputs == boundInputs) &&
+                  ::IsUpToDate(config_, version->config)) {
 
 #if 0
-                notify(NFY_NOTICE,
-                       "${name}: ReuseOrMakeAndUpdate (reusing %s)",
-                       version->GetRef().c_str());
-                notify(NFY_NOTICE, "         boundinputs:");
-                for (const auto & bi : boundInputs) {
-                    notify(NFY_NOTICE, "             %s", bi.c_str());
-                }
-                notify(NFY_NOTICE, "         version inputs:");
-                for (std::vector<std::string>::const_iterator iv =
-                     version->inputs.begin();
-                     iv != version->inputs.end(); ++iv) {
-                    notify(NFY_NOTICE, "             %s", iv->c_str());
-                }
+                  notify(NFY_NOTICE,
```

Comment:
besides being a spacing/tab error, do we really want to carry this code block?

Reply:
I just further indented the existing formatting. How would you prefer it? I try not to change the existing code much, in hopes of keeping it consistent.

Reply:
Cleaned up some of the spacing to reduce the noise.